openid-caep-interoperability-profile-1_0.md

```diff
@@ -217,21 +217,26 @@ All events MUST be signed using the `RS256` algorithm using a minimum of 2048-bi
 ## OAuth Service
 
 ### Authorization Server
-* MAY distribute discovery metadata (such as the authorization endpoint) via the metadata document as specified in [RFC8414]{{RFC8414}}
-* MUST support at least one of the following to obtain a short-lived access token. For example, a short lived access token could be defined as one in which the value of the `exp` claim is not longer than 60 mins after `nbf` claim. Please refer to Access token lifetimes in the security considerations of {{FAPI}} for additional considerations. 
+
+An OAuth {{RFC6749}} Authorization Server issues access tokens. In the context of this profile, the Authorization Server that issues access tokens can be a separate entity than the SSF Transmitter.
```

Comment:
An OAuth {{RFC6749}} Authorization Server *that* issues access tokens. (the word "that" is missing there)

Reply:
this is an informational line describing what an OAuth AS is. i don't think that makes sense gramatically. 
We could rephrase it to align with your thoughts here